packages/cli/src/ui/commands/compressCommand.test.ts

```diff
@@ -90,6 +91,22 @@ describe('compressCommand', () => {
     expect(context.ui.setPendingItem).toHaveBeenNthCalledWith(2, null);
   });
 
+  it('should pass instruction to tryCompressChat', async () => {
+    const compressedResult = {
+      originalTokenCount: 200,
+      newTokenCount: 100,
+    };
+    mockTryCompressChat.mockResolvedValue(compressedResult);
+
+    await compressCommand.action!(context, 'test instruction');
+
+    expect(mockTryCompressChat).toHaveBeenCalledWith(
+      expect.stringMatching(/^compress-\d+$/),
```

Comment:
specify where the new instructions occur in the output rather than just that they occur. this could be achieved with a snapshot test. you can see other examples of text snapshot tests including one for the system prompt.